Library/Homebrew/migrator.rb

```diff
@@ -166,12 +191,12 @@ def fix_tabs
 
   sig { returns(T::Boolean) }
   def from_same_tap_user?
-    formula_tap_user = formula.tap.user if formula.tap
+    formula_tap_user = formula.tap&.user
     old_tap_user = nil
 
     new_tap = if old_tap
```

Comment:
```suggestion
    new_tap = if (old_tap = self.old_tap)
```
and then can remove `T.must`s below